### DISCUSSION TITLE
Telemetry Data From ELRS AirPort

### DISCUSSION THREAD

**Discussion by koklimabc** [stronnag/mwptools#175] [Q&A] [answered]
![need_help](https://github.com/stronnag/mwptools/assets/2577622/41568c11-3256-4f88-85f3-acfabf75c40b)

Sorry to tamper you again:), 
I does enable elrs airport function on both rx & tx with baud 4800 and connect into mwptools (chown username ttyUSB0),
but it show out nothing. 
I attached along pictures for port setting through Inav configurator, Does it correct?
FYI, Im using Bandit Micro & BR3 Rx both 
915Mhz
Hope you could help me to fix :)

**Comment by stronnag** [comment, marked as answer]
mwp works with transparent serial. It does not care about the hardware between it and the FC.
From discussions with others, I'm told Airport is at best "unreliable". In particular it has little or no hardware buffering, so things like `MSP_BOXNAMES` is doomed to fail.

If would also help if you were to read the [Help and Support](https://www.daria.co.uk/mwptools/mwp_support/) section of the user guide and provide the expected data, in particular the `mwp_stderr*` log file.

**Comment by stronnag** [comment]
It is also the case (IIRC), that you may not get LTM until you arm the FC.
Please also capture the data stream with `mwp --raw-log`

**Comment by stronnag** [comment]
And if you want MSP pre-arm and LTM post-arm, you need to (a) enable both and(b) set the same baud rate for both.

**Comment by koklimabc** [comment]
Ha, After half-day of tried settings with airport on both Tx and Rx, my remote controller won't stick any reaction onto AER at all, I decided not to switch on AirPort function anymore. Your are rights , elrs airport are so flawness and unreliable for this momnet. 

mwp is able trully connect to FlightControler (through /dev/ttyACM0) and Remote Tx for sure without problem now :)

Below is Tx 
![Screenshot from 2024-04-28 20-16-00](https://github.com/stronnag/mwptools/assets/2577622/85f84603-afba-4d2f-9efe-9cad00c33724)

Below is FC
![Screenshot from 2024-04-28 21-54-33](https://github.com/stronnag/mwptools/assets/2577622/d71dfbc7-3d82-4935-912f-79c955461b81)
